scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RunProcessor.java

```diff
@@ -87,7 +87,9 @@ public Platform getPlatformType() {
     Stream<Builder> standard =
         Stream.of(
             new Builder(Platform.ILLUMINA, "default", DefaultIllumina::create),
-            new Builder(Platform.PACBIO, "default", DefaultPacBio::create));
+            new Builder(Platform.PACBIO, "default", DefaultPacBio::create),
+            new Builder(Platform.PROMETHION, "default", DefaultPromethion::create),
```

Comment:
Rather than have two platforms that are `"default"`, make these both `OXFORDNANOPORE` and make one `"promethion"` and the other `"minion"`.